UI/Source/Common/mac/ModelCollectionViewController.swift

```diff
@@ -0,0 +1,385 @@
+import AppKit
+import Pilot
+
+/// Common view controller to support a scrollable views of a ModelCollection.
+open class ModelCollectionViewController: NSViewController {
```

Comment:
should this be `public` rather than `open`? or do you want outside modules to subclass it?